.github/ISSUE_TEMPLATE/new-release-template.md

```diff
@@ -7,30 +7,41 @@ assignees: comcmaho, amdeel, davidmrdavid, bachuv
 
 ---
 
-**Prep Release: (assigned to:)**
-_Due: <3-business-days-before-release>_
-- [ ] Merge all features and fixes into the `dev` branch
-- [ ] Publish updated versions of DurableTask.Core and DurableTask.AzureStorage to app-service MyGet feed
-- [ ] Update dependencies and increment extension version in the `dev` branch.
-- [ ] Publish signed version of Microsoft.Azure.WebJobs.Extensions.DurableTask to app-service MyGet feed
-- [ ] Close out or punt remaining GitHub issues for the current milestone
+**Prep DTFx Release: (assigned to:)**
+_Due: <2-3-business-days-before-release>_
+- [ ] Increment updated DTFx packages (either DT-AzureStorage only or DT-AzureStorage, DT-Core, and DT-Redis)
+- [ ] Delete DTFx test packages from myget.
+- [ ] Update OneBranch durabletask repo, run signing pipeline (using these notes:  [Durable Task Framework Build Pipeline] (https://microsoft.sharepoint.com/teams/AzureWebjobs/_layouts/OneNote.aspx?id=%2Fteams%2FAzureWebjobs%2FSiteAssets%2FAzureWebjobs%20Notebook&wd=target%28Planning%2FFunctions%20Post-GA.one%7CA43CF112-7272-481A-B23E-9AA5CA8EEE06%2FDurable%20Task%20Framework%20Build%20Pipeline%7CD0946823-6FB0-44E3-A57F-E252617B69CD%2F%29
```

Comment:
are we good to share one-notes on the public repo? Just checking!

Reply:
I think permissions should block people who shouldn't have access from access, but it's still a valid point.

Reply:
As an aside, it might be worth considering redoing the DTFx release pipeline to something simpler, like what we do for the Durable extension w/Azure DevOps. The whole thing with using the msazure OneBranch and sub-modules (thus requiring all these steps in the OneNote) seems unnecessarily complicated. Just my $.02.

Reply:
I'm 100% on board for simplifying the DTFx release pipeline. @cgillum, I may look into what you did for your sql provider repo to see if we can make something simpler for the broader DTFx repo.